processing/src/main/java/org/apache/druid/segment/index/BitmapColumnIndex.java

```diff
@@ -38,10 +38,7 @@ public interface BitmapColumnIndex
   /**
    * Returns an estimated cost for computing the bitmap result.
    */
-  default int estimatedComputeCost()
-  {
-    return Integer.MAX_VALUE;
-  }
+  int estimatedComputeCost();
```

Comment:
i know this isn't new in this PR, but i feel like maybe the javadoc should mention that the estimated cost should be related to the number of bitmap operations that need to be performed to compute the filter bitmap

Reply:
added more explanation on this